code/modules/jobs/job_types/wasteland.dm

```diff
@@ -374,6 +380,8 @@ Raider
 		/obj/item/gun/ballistic/revolver/m29/snub = 2,
 		/obj/item/storage/belt/holster = 1,
 		/obj/item/ammo_box/m44 = 3,
+		/obj/item/radio/headset = 1,
+		/obj/item/radio/headset = 1,
```

Comment:
Two radio headsets here, don't know if it's intended or a bug, just saying.

Reply:
fixed, if you see anything else don't hesitate to say.